priv/templates/_admin_edit_options.acl_collaboration_group.tpl

```diff
@@ -64,4 +64,20 @@
         {_ Descending _}
     </label>
 </div>
+{% if m.acl.user.o.hasusergroup[1].id == m.rsc.acl_user_group_community_librarian.id or m.acl.use.mod_admin %}
```

Comment:
From what I see in the installed ACL rules `acl_user_group_community_librarian` have the permission to use `mod_admin`, so this could be:
```suggestion
{% if m.acl.use.mod_admin %}
```

P.s. checking usergroup by `hasusergroup` edges like is not working for sure, as users can have multiple usergroups and they might be ordered in a way that puts `acl_user_group_community_librarian` in a position different than `1`